thorvg/examples/thorvg-example/main/display_main.c

```diff
@@ -0,0 +1,221 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include <stdlib.h>
+#include <dirent.h>
+#include "esp_log.h"
+#include "esp_err.h"
+#include "esp_check.h"
+#include "esp_lcd_panel_ops.h"
+#include "bsp/esp-bsp.h"
+#include "driver/ppa.h"
+#include "thorvg_capi.h"
+
+static const char *TAG = "example";
+
+static void capi_loop_task(void *arg);
+static esp_err_t capi_create_lottie(bsp_lcd_handles_t *lcd_panel);
+static esp_err_t argb888_to_rgb565_ppa(ppa_client_handle_t ppa_handle, const uint32_t *in, uint16_t *out);
+
+#define LOTTIE_SIZE_HOR     320
+#define LOTTIE_SIZE_VER     320
+
+void app_main(void)
+{
+    static bsp_lcd_handles_t lcd_panel;
+
+    bsp_spiffs_mount();
+
+    /* Initialize display */
+    bsp_display_new_with_handles(NULL, &lcd_panel);
+
+    bsp_display_backlight_on();
+
+    BaseType_t res = xTaskCreate(capi_loop_task, "thorvg task", 60 * 1024, (void *)&lcd_panel, 5, NULL);
+    if (res != pdPASS) {
+        ESP_LOGE(TAG, "Create thorvg task fail!");
+    }
+}
+
+void capi_loop_task(void *arg)
+{
+    bsp_lcd_handles_t *lcd_panel = (bsp_lcd_handles_t *)arg;
+
+    while (1) {
+        capi_create_lottie(lcd_panel);
+        vTaskDelay(pdMS_TO_TICKS(2000));
+    }
+
+    vTaskDelete(NULL);
+}
+
+static esp_err_t capi_create_lottie(bsp_lcd_handles_t *lcd_panel)
+{
+    esp_err_t ret = ESP_OK;
+    Tvg_Result tvg_res = TVG_RESULT_SUCCESS;
+    Tvg_Result tvg_engine = TVG_RESULT_INSUFFICIENT_CONDITION;
+
+    static uint32_t reac_color = 0x00;
+
+    char *src = "/spiffs/lolo_walk.json";
```

Comment:
```suggestion
    const char *src = "/spiffs/lolo_walk.json";
```

should cause a -Wwrite-strings warning otherwise.